Add provider configuration for linux_64

vicentebolea opened the pull request.

vicentebolea committed:
Add provider configuration for linux_64

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

vicentebolea committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.09.17.39.06

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10
